Gruntfile.js

```diff
@@ -99,7 +99,11 @@ module.exports = function (grunt) {
     grunt.registerTask('compile:js', function (fullCompile) {
         grunt.task.run(['clean:js', 'compile:inlineSvgs']);
 
-        grunt.task.run(['concurrent:requireJS', 'copy:javascript']);
+        if (!options.isDev) {
```

Comment:
it's just not used at all. we can still build them, but you'll never load them in dev